pkg/garbagecollector/announcement_handlers.go

```diff
@@ -0,0 +1,74 @@
+package garbagecollector
+
+import (
+	"context"
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/rs/zerolog/log"
```

Comment:
we should not be using zerologger, use `"github.com/openservicemesh/osm/pkg/logger"` instead

Reply:
Changed to use `"github.com/openservicemesh/osm/pkg/logger"` instead